ui-toolkit/modals.md

```diff
@@ -28,7 +28,7 @@ category: UI toolkit
 <div class="content-50 content-first">
   <h4 id="examples">Examples of when to use modals</h4>
   <ul>
-    <li>Confirm an action that can’t be undone, such as “Are you sure you want to delete this</li>
+    <li>Confirm an action that can’t be undone, such as “Are you sure you want to delete this?</li>
```

Comment:
Needs a closing quotation mark after the question mark.

Reply:
Whoops, just commented on the prev PR removing before realizing there was one adding it back in.

Reply:
oh, I forgot to merge this, but actually this file isn't even the one being used to generate the real design manual page, it is a duplicate in an old now-unused folder.